v10.8 Changelog (#7888)
mattermost/docs: source/about/mattermost-v10-changelog.md

```{include} common-esr-support-upgrade.md
```

(release-v10.8-feature-release)=
## Release v10.8 - [Feature Release](https://docs.mattermost.com/about/release-policy.html#release-types)

**Release day: 2025-05-16**

### Important Upgrade Notes

>  - New tables ``AccessControlPolicies`` and ``AcessControlPolicyHistory`` will be created. The migration is fully backwards-compatible, non-locking, and zero downtime is possible.

Reviewer: > and zero downtime is possible.

I find this to be ambiguous. Will there be zero down-time or not?  If it's merely "possible" to have zero-down time, is there something an admin is supposed to do to make "possible" a certainty?

```{Important}
If you upgrade from a release earlier than v10.7, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
```

### Improvements

#### User Interface (UI)
 - Pre-packaged Calls plugin version [v1.7.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v1.7.1).
 - Pre-packaged Metrics plugin version [v0.6.0](https://github.com/mattermost/mattermost-plugin-metrics/releases/tag/v0.6.0).
 - Added an improved channel menu. 
 - Updated email notification settings to provide clearer wording and descriptions for both batched and non-batched scenarios. The settings dialog now reflects the selected status more accurately in both collapsed and expanded views, enhancing consistency and usability.
 - Added the ability to display the nickname or full name in Threads based on settings.
 - Improved the error message for failed file copies. 

#### Administration
 - Added Custom Profile Attribute field type, visibility, and related options in **System Console -> System Properties**.

>  - Introduced Premium SKU in code and removed usages of E10 and E20 licenses.

Reviewer 2: @amyblais the new sku is going out in the next release. We should update to just: 

```suggestion
 - Removed support for legacy skus E10 and E20 licenses.
```

Reviewer 3: I'll defer to @BillAnderson304 and the Product team on this. My suggestion is to add a CTA to talk to a Mattermost expert.
